bigml/anomaly.py

```diff
@@ -101,9 +100,9 @@ def __init__(self, anomaly, api=None):
                                         "Score will"
                                         " not be available")
                     else:
-                        default_depth =  (
-                            2 * (0.5772156649 + 
-                                 math.log(self.sample_size - 1) -
+                        default_depth = (
+                            2 * (0.5772156649 + \
```

Comment:
is this magic number 0.57... something we could give a name to? (to define a constant for self-documentation purporses).